apparmor.d/groups/apps/calibre

```diff
@@ -98,27 +142,30 @@ profile calibre @{exec_path} {
   owner /tmp/calibre_*_tmp_*/{,**} rw,
   owner /tmp/calibre-*/{,**} rw,
   owner /tmp/[0-9]*-*/ rw,
-  owner /tmp/[0-9]*-*/** rwl -> /tmp/[0-9]*-*/**,
+  owner /tmp/[0-9]*-*/** rwl,
+#  owner /tmp/[0-9]*-*/** rwl -> /tmp/[0-9]*-*/**,  # newer AA version
```

Comment:
Or bug: https://lists.snapcraft.io/archives/apparmor/2019-October/012050.html

Don't know why it passes the test.

Reply:
I will keep this in mind. If more recent AA version support it, they could be a system to change the profile accordingly during the build time.